mbedtls/src/bignum/mod.rs

```diff
@@ -173,7 +173,7 @@ impl Mpi {
             mpi_write_string(
                 &self.inner,
                 radix,
-                buf.as_mut_ptr() as *mut i8,
+                buf.as_mut_ptr() as *mut _,
```

Comment:
By curiosity, why is this needed?

Reply:
OK Let me check. I sync these few lines of code  with master version
This `i8` to `_` changes may be not necessary here. 
When I first testing, I used the a copy of master version ct.sh which will also testing on aarch64 platform. So in aarch64, it need to be u8.
So `_` could be automatically infer to i8 or 8u based on different architecture.

Although 0.7 does not include support `aarch64`.  If you want I can reverse this modification.
I feel it's fine to just leave change here.

Reply:
It's fine, master has the same. Thanks.